javalib/src/main/scala/java/io/FileOutputStream.scala

```diff
@@ -16,6 +16,7 @@ import scala.scalanative.windows.winnt.AccessRights._
 
 class FileOutputStream(fd: FileDescriptor, file: Option[File] = None)
```

Comment:
The default parameter is now redundant with the overload, and should be removed.